### PR TITLE
feature/ejp-multiple-links-bug-5276

### PR DESCRIPTION
**Pull request by tiandavis**
## Summary - [BZ-5276](https://thirdiron.atlassian.net/browse/BZ-5276)

Refactored EJP to get target by ISSN instead of Title name.

## Deploy Precautions

- None

### CODE REVIEW

File: tests/unit/models/360-core.js [hunk 1]
```diff
@@ -300,41 +300,20 @@ describe("SerSol 360 Core Model >", function() {
...
-      $("div[ui-view='searchResults']").find(".results-title:contains('" + journalName + "')")[0].innerHTML = journalName.toLowerCase();
-
-      var target = serSol360Core.getTarget(title);
-
-      expect(target).toBeDefined();
-      expect(journalName).toEqual("The New England journal of medicine");
-      expect(target.innerHTML).toContain("the new england journal of medicine");
-
-      $("div[ui-view='searchResults']").find(".results-title:contains('" + journalName.toLowerCase() + "')")[0].innerHTML = title.title;
+      expect(target.innerHTML).toContain("0028-4793");
```
Comment (karlbecker) [inline tests/unit/models/360-core.js]: Do you think we need to search for the ISSN both with and without a dash, or has it been very consistently with a dash?

File: src/primo/browzine-primo-adapter.js [hunk 2]
```diff
@@ -465,7 +465,6 @@ browzine.primo = (function() {
     showArticleBrowZineWebLinkText: showArticleBrowZineWebLinkText,
     showDirectToPDFLink: showDirectToPDFLink,
     showPrintRecords: showPrintRecords,
-    isFiltered: isFiltered,
```
Comment (karlbecker) [inline src/primo/browzine-primo-adapter.js]: Ooh I see, this was already defined up higher in this object 👍

File: tests/unit/models/360-core.js (hunk 1 above)
Comment (tiandavis) [reply tests/unit/models/360-core.js]: Very consistently with a dash, seems to be the standard.

File: src/primo/browzine-primo-adapter.js (hunk 2 above)
Comment (tiandavis) [reply src/primo/browzine-primo-adapter.js]: Yep. Code cleanup.

**Comment by tiandavis** [comment]
Got the regression in here @karlbecker! 
https://github.com/thirdiron/browzine-discovery-service-adapters/pull/57/commits/2cc99a0b6d4e44cce3f8e8ee34cf0be16c9a1ba3

Shouldn't be seeing that bug again! 🎉

**Pull request merged**